src/alg_utils.jl

```diff
@@ -31,7 +31,7 @@ ismultistep(alg::ETD2) = true
 
 isadaptive(alg::OrdinaryDiffEqAlgorithm) = false
 isadaptive(alg::OrdinaryDiffEqAdaptiveAlgorithm) = true
-isadaptive(alg::OrdinaryDiffEqCompositeAlgorithm) = isadaptive(alg.algs[1])
+isadaptive(alg::OrdinaryDiffEqCompositeAlgorithm) = isadaptive(alg.algs[alg.current_alg])
```

Comment:
This is used to choose the default for adaptivity on/off. I think it should be `all(isadaptive(...))`?